apps/mull-ui/src/app/app.tsx

```diff
@@ -41,6 +44,14 @@ export const App = () => {
             <EventPage event={dummyEvent} prevPage={'Review'} />
           </div>
         </Route>
+        {/* Without this outter Route component, the inner components would show on all routes that are not matched */}
+        <Route exact path={[`${ROUTES.DISCOVER}`, `${ROUTES.UPCOMING}`, `${ROUTES.MYEVENTS}`]}>
+          <SwipeableRoutes>
+            <Route path={ROUTES.DISCOVER} component={SubNavigationBar} />
+            <Route path={ROUTES.UPCOMING} component={SubNavigationBar} />
+            <Route path={ROUTES.MYEVENTS} component={SubNavigationBar} />
+          </SwipeableRoutes>
+        </Route>
```

Comment:
I don't know about you @r-saba but I really don't like how the sub-navbar slides along with the page. It makes it really confusing. I experimented a bit and found a solution. Let me know what you think.
```suggestion
        <Route exact path={[`${ROUTES.DISCOVER}`, `${ROUTES.UPCOMING}`, `${ROUTES.MYEVENTS}`]}>
          <SubNavigationBar />
          <SwipeableRoutes>
            <Route path={ROUTES.DISCOVER} component={() => <div>DISCOVER!</div>} />
            <Route path={ROUTES.UPCOMING} component={() => <div>UPCOMING!</div>} />
            <Route path={ROUTES.MYEVENTS} component={() => <div>MYEVENTS!</div>} />
          </SwipeableRoutes>
        </Route>
```

Reply:
Oh good point, didn't consider that we probably don't want the subnavigation to slide as well